osmosis-1/osmosis.zone_config.json

```diff
@@ -391,6 +422,30 @@
         "pretty_name": "Tron",
         "symbol_suffix": "trx",
         "representative_asset": "sun"
+      },
+      {

```

Comment:
Please remove this edit, since it's not being used correctly. (meant to indicate assets from evm chains)

Reply:
Removed, please check it out again